docs/advanced/cri.md

```diff
@@ -101,8 +101,12 @@ By default, `CRI-O` uses `cgroupfs` as a cgroup driver manager. If you want to s
 # Cgroup management implementation used for the runtime.
 cgroup_manager = "systemd"
 ```
+:::note
+The image should be updated if you are on ARM platform and the image you are using is not a multi-arch image.
+:::
 
-*NOTE: the `pause` image should be updated if you are on ARM platform and the `pause` image you are using is not a multi-arch image. To set the pause image, update the `CRI-O` config file:*
+
+To set the pause image, update the `CRI-O` config file:
```

Comment:
Here, I guess two lines were combined by mistake into the note section, so I've separated them as well